variables.tf

```diff
@@ -29,7 +29,7 @@ variable "description" {
 
 variable "region" {
   description = "update"
-  default = "US"
+  default     = "US"
 }
 
 variable "expiration" {
```

Comment:
Is "update" our keyword for users to replace? Should we add a description of what the variable is and sample values?

Reply:
yes, in the next PR the descriptions should be updated